docs/assessment.md

```diff
@@ -106,6 +106,17 @@ For practice exams it is often desirable to make a *multiple instance* assessmen
 ## Enabling group work for collaborative assessments
 
 By default, assessment instances are tied to only one user. By setting `groupWork: true`, multiple students will be able to work on the same assessment instance.
+Instructor should also set information about group size constraints and student permissions in the json format. Example:
+```json
+{
+        "groupWork": true,
+        "groupMaxSize": 6,
+        "groupMinSize": 2,
+        "studentGroupCreate": true,
+        "studentGroupJoin": true,
+        "studentGroupLeave": true,
+}
+```
```

Comment:
Can we add a table listing all the options here, their types, default values, and descriptions? This would be similar to the tables of options here: https://prairielearn.readthedocs.io/en/latest/elements/#customizations